docs/modules/reference/pages/model.adoc

```diff
@@ -1,17 +1,17 @@
 = Model
-:imagesdir: ../assets/images 
+:imagesdir: ../assets/images
 
```

Comment:
It would be nice to add a page description here, like we have on other pages. As it is, I'm not sure what "model" refers to.